utils/extensions/models/message_status.ts

```diff
@@ -31,14 +30,10 @@ export class MessageStatusDeletableModel extends MessageStatusModelBase {
    */
   public findAllVersionsByModelId(
     modelId: NonEmptyString
-  ): DocumentDbUtils.IResultIterator<RetrievedMessageStatus> {
-    return DocumentDbUtils.findAllVersionsByModelId(
-      this.dbClient,
-      this.collectionUri,
-      MESSAGE_STATUS_MODEL_ID_FIELD,
-      modelId,
-      MESSAGE_STATUS_MODEL_PK_FIELD,
-      modelId
-    );
+  ): AsyncIterator<ReadonlyArray<t.Validation<RetrievedMessageStatus>>> {
+    return this.getQueryIterator({
```

Comment:
can't we use findAllVersionsByModelId in utils?

Reply:
fixed in [#9f1381a](https://github.com/pagopa/io-functions-admin/pull/65/commits/9f1381a01f8be2a475463f72e92d9eaa5461f8dd)